src/main/java/core/basesyntax/orderprocessing/OrdersReader.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.orderprocessing;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+import core.basesyntax.customexceptions.BadFileFormatting;
+import core.basesyntax.entries.FruitPack;
+import core.basesyntax.entries.Order;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrdersReader {
```

Comment:
```suggestion
public class FileReader {
```

Reply:
Can't rename it that way because I use java.io.FileReader in this class